spec/controllers/users_controller_spec.rb

```diff
@@ -95,6 +95,23 @@
       end
     end
 
+    describe '#show_update_email' do
+      it 'renders the update email page' do
+        get :show_update_email, :params => { :user_id => user_id }
+
+        expect(response).to have_http_status(:success)
+        expect(subject).to render_template(:show_update_email)
+      end
+    end
+    
+    describe '#update_email' do
```

Comment:
There are 4 logical branches in that controller, therefore I would expect at least 4 tests + 1 where `update_user_email_form` param is missing.